ramips: mt7621: add support for Confiabits MT7621 v1

LuisMitaHL opened the pull request:
Working:

- UART
- Ethernet WAN
- Ethernet LAN1/2/3
- Wi-Fi 2.4GHz
- Wi-Fi 5GHz
- LEDs
- SPI Flash
- Correct MAC addresses
- Reset & WPS buttons

LuisMitaHL committed:
ramips: mt7621: add support for Confiabits MT7621 v1

Hardware:
 - SoC: MediaTek MT7621DAT
 - Flash: 16 MiB cFeon
 - RAM: 128 MiB MT7621DAT
 - WLAN: 2.4 GHz (MT7603EN), 5 GHz (MT7613BEN 802.11ac)
 - Ethernet: 1x 10/100/1000 Mbps WAN, 3x 10/100/1000 LAN (MT7621DAT)
 - Buttons: 1 Reset button, 1 WPS button
 - LEDs: 5x Green (POWER/WAN/2.4G/5G/WPS), 1x Red (WAN)
 - Serial console: unpopulated header, 57600 8n1
 - Power: 12 VDC, 1 A

MAC:
LAN MAC: label mac (eeprom@4)
WAN MAC: label mac (eeprom@4)
2.4G MAC: label mac (eeprom@4)
5G MAC: label mac + 1 (eeprom@8004)

Installation:
The stock firmware is OpenWrt-based. If you can reach LuCI or SSH, just use the sysupgrade image
with the 'Keep settings' option turned off.

Signed-off-by: Luis Mita <luis@luismita.com>
Link: https://github.com/openwrt/openwrt/pull/17534
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

csharper2005 commented on target/linux/ramips/dts/mt7621_confiabits_mt7621-v1.dts:
```diff
@@ -0,0 +1,223 @@
+		compatible = "mediatek,mt76";
+		reg = <0x0000 0 0 0 0>;
+		ieee80211-freq-limit = <5000000 6000000>;
+		mediatek,disable-radar-background;
```
Why do you need `mediatek,disable-radar-background;`? AFAIK it's for mt7915.

LuisMitaHL replied:
Yes, I checked mt76 driver and this line is useless for mt7613. Line deleted

csharper2005 commented on target/linux/ramips/dts/mt7621_confiabits_mt7621-v1.dts:
```diff
@@ -0,0 +1,223 @@
+&pcie {
+	status = "okay";
+};
+
```
Line can be dropped.

LuisMitaHL replied:
done

csharper2005 commented on target/linux/ramips/dts/mt7621_confiabits_mt7621-v1.dts:
```diff
@@ -0,0 +1,223 @@
+
+	};
+};
+
```
Line can be dropped.

csharper2005 commented on target/linux/ramips/dts/mt7621_confiabits_mt7621-v1.dts:
```diff
@@ -0,0 +1,223 @@
+			function = LED_FUNCTION_WPS;
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+
```
Lines can be dropped.

LuisMitaHL replied:
done

csharper2005 commented on target/linux/ramips/dts/mt7621_confiabits_mt7621-v1.dts:
```diff
@@ -0,0 +1,223 @@
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
```
Line can be dropped.

LuisMitaHL replied:
done

csharper2005 reviewed (commented):
Please, add MAC address table to the commit message. And sort the LEDs by their GPIO numbers.

csharper2005 commented:
> Done

Looks good to me. Thank you.

hauke commented on target/linux/ramips/dts/mt7621_confiabits_mt7621-v1.dts:
```diff
@@ -0,0 +1,217 @@
+
+		led-wlan5 {
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_WLAN;
+			function-enumerator = <5>;
```
Please use this instead:
```suggestion
			function = LED_FUNCTION_WLAN_5GHZ;
```

LuisMitaHL replied:
Done

LuisMitaHL commented on target/linux/ramips/dts/mt7621_confiabits_mt7621-v1.dts:
```diff
@@ -0,0 +1,217 @@
+		led-wlan24 {
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_WLAN;
+			function-enumerator = <2>;
```
Done

namiltd commented:
Could you check this patch and paste the syslog?: https://github.com/openwrt/openwrt/issues/17764#issuecomment-2625071946

LuisMitaHL commented:
> Could you check this patch and paste the syslog?: [#17764 (comment)](https://github.com/openwrt/openwrt/issues/17764#issuecomment-2625071946)

Here you go @namiltd 

<details><summary>Details</summary>
<pre>

root@OpenWrt:~# logread
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    0.000000] Linux version 6.6.74 (looper@amelia) (mipsel-openwrt-linux-musl-gcc (OpenWrt GCC 13.3.0 r28444-1aae1dfaf2) 13.3.0, GNU ld (GNU Binutils) 2.42) #0 SMP Thu Jan 30 12:35:23 2025
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] SoC Type: MediaTek MT7621 ver:1 eco:3
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] printk: bootconsole [early0] enabled
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] CPU0 revision is: 0001992f (MIPS 1004Kc)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] MIPS: machine is Confiabits MT7621 v1
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Initrd not found or empty - disabling initrd
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] VPE topology {
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] cpu0:0 flags = 0,
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] cpu0:1 flags = 1,
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] cpu1:0 flags = 1,
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] cpu1:1 flags = 1
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    0.000000] 2,2} total 4
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Primary instruction cache 32kB, VIPT, 4-way, linesize 32 bytes.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Primary data cache 32kB, 4-way, PIPT, no aliases, linesize 32 bytes
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    0.000000] MIPS secondary cache 256kB, 8-way, linesize 32 bytes.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Zone ranges:
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000]   Normal   [mem 0x0000000000000000-0x0000000007ffffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Movable zone start for each node
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Early memory node ranges
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000]   node   0: [mem 0x0000000000000000-0x0000000007ffffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Initmem setup node 0 [mem 0x0000000000000000-0x0000000007ffffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] percpu: Embedded 12 pages/cpu s18816 r8192 d22144 u49152
Thu Jan 30 12:35:29 2025 kern.debug kernel: [    0.000000] pcpu-alloc: s18816 r8192 d22144 u49152 alloc=12*4096
Thu Jan 30 12:35:29 2025 kern.debug kernel: [    0.000000] pcpu-alloc: [0] 0 [0] 1 [0] 2 [0] 3
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    0.000000] Kernel command line: console=ttyS0,57600 rootfstype=squashfs,jffs2
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Dentry cache hash table entries: 16384 (order: 4, 65536 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Inode-cache hash table entries: 8192 (order: 3, 32768 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Writing ErrCtl register=00002ce0
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Readback ErrCtl register=00002ce0
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 32480
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] mem auto-init: stack:off, heap alloc:off, heap free:off
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] Memory: 117684K/131072K available (8035K kernel code, 654K rwdata, 1708K rodata, 1240K init, 219K bss, 13388K reserved, 0K cma-reserved)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] SLUB: HWalign=32, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] rcu: Hierarchical RCU implementation.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000]       Tracing variant of Tasks RCU enabled.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] rcu: RCU calculated value of scheduler-enlistment delay is 10 jiffies.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] NR_IRQS: 256
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] rcu: srcu_init: Setting srcu_struct sizes based on contention.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000000] clocksource: GIC: mask: 0xffffffffffffffff max_cycles: 0xcaf478abb4, max_idle_ns: 440795247997 ns
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.000004] sched_clock: 64 bits at 880MHz, resolution 1ns, wraps every 4398046511103ns
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.016042] Calibrating delay loop... 586.13 BogoMIPS (lpj=2930688)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.088318] pid_max: default: 32768 minimum: 301
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.106289] Mount-cache hash table entries: 1024 (order: 0, 4096 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.120706] Mountpoint-cache hash table entries: 1024 (order: 0, 4096 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.143152] cpu0:0 flags = 0,
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.143181] cpu0:1 flags = 1,
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.149162] cpu1:0 flags = 1,
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.155141] cpu1:1 flags = 1
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.168583] RCU Tasks Trace: Setting shift to 2 and lim to 1 rcu_task_cb_adjust=1 rcu_task_cpu_ids=4.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.187414] rcu: Hierarchical SRCU implementation.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.196788] rcu:  Max phase no-delay instances is 1000.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.208001] smp: Bringing up secondary CPUs ...
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.217844] Primary instruction cache 32kB, VIPT, 4-way, linesize 32 bytes.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.217873] Primary data cache 32kB, 4-way, PIPT, no aliases, linesize 32 bytes
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    0.217888] MIPS secondary cache 256kB, 8-way, linesize 32 bytes.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.217935] CPU1 revision is: 0001992f (MIPS 1004Kc)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.277468] Synchronize counters for CPU 1: done.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.339279] Primary instruction cache 32kB, VIPT, 4-way, linesize 32 bytes.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.339302] Primary data cache 32kB, 4-way, PIPT, no aliases, linesize 32 bytes
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    0.339314] MIPS secondary cache 256kB, 8-way, linesize 32 bytes.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.339343] CPU2 revision is: 0001992f (MIPS 1004Kc)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.398482] Synchronize counters for CPU 2: done.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.458913] Primary instruction cache 32kB, VIPT, 4-way, linesize 32 bytes.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.458936] Primary data cache 32kB, 4-way, PIPT, no aliases, linesize 32 bytes
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    0.458949] MIPS secondary cache 256kB, 8-way, linesize 32 bytes.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.458981] CPU3 revision is: 0001992f (MIPS 1004Kc)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.518073] Synchronize counters for CPU 3: done.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.577677] smp: Brought up 1 node, 4 CPUs
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.591837] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.611341] futex hash table entries: 1024 (order: 3, 32768 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.628927] pinctrl core: initialized pinctrl subsystem
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.642833] NET: Registered PF_NETLINK/PF_ROUTE protocol family
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.657340] /pinctrl: Fixed dependency cycle(s) with /pinctrl/pinctrl0
Thu Jan 30 12:35:29 2025 kern.debug kernel: [    0.674222] FPU Affinity set after 11720 emulations
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.686945] clocksource: Switched to clocksource GIC
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.706854] NET: Registered PF_INET protocol family
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.716742] IP idents hash table entries: 2048 (order: 2, 16384 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.731901] tcp_listen_portaddr_hash hash table entries: 512 (order: 0, 4096 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.748424] Table-perturb hash table entries: 65536 (order: 6, 262144 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.763779] TCP established hash table entries: 1024 (order: 0, 4096 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.779035] TCP bind hash table entries: 1024 (order: 2, 16384 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.793229] TCP: Hash tables configured (established 1024 bind 1024)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.806752] MPTCP token hash table entries: 256 (order: 0, 4096 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.821342] UDP hash table entries: 256 (order: 1, 8192 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.834232] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes, linear)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.849010] NET: Registered PF_UNIX/PF_LOCAL protocol family
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.860255] PCI: CLS 0 bytes, default 32
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.871591] workingset: timestamp_bits=14 max_order=15 bucket_order=1
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.886268] squashfs: version 4.0 (2009/01/31) Phillip Lougher
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.897775] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.922397] mt7621_gpio 1e000600.gpio: registering 32 gpios
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.933810] mt7621_gpio 1e000600.gpio: registering 32 gpios
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.945288] mt7621_gpio 1e000600.gpio: registering 32 gpios
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.957000] mt7621-pci 1e140000.pcie: host bridge /pcie@1e140000 ranges:
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.970243] mt7621-pci 1e140000.pcie:   No bus range found for /pcie@1e140000, using [bus 00-ff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    0.987761] mt7621-pci 1e140000.pcie:      MEM 0x0060000000..0x006fffffff -> 0x0060000000
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.003943] mt7621-pci 1e140000.pcie:       IO 0x001e160000..0x001e16ffff -> 0x0000000000
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.376948] mt7621-pci 1e140000.pcie: pcie2 no card, disable it (RST & CLK)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.390734] mt7621-pci 1e140000.pcie: PCIE0 enabled
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.400376] mt7621-pci 1e140000.pcie: PCIE1 enabled
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.410232] PCI coherence region base: 0x60000000, mask/settings: 0xf0000002
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.424300] mt7621-pci 1e140000.pcie: PCI host bridge to bus 0000:00
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.436851] pci_bus 0000:00: root bus resource [bus 00-ff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.447700] pci_bus 0000:00: root bus resource [mem 0x60000000-0x6fffffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.461352] pci_bus 0000:00: root bus resource [io  0x0000-0xffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.473672] pci 0000:00:00.0: [0e8d:0801] type 01 class 0x060400
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.485561] pci 0000:00:00.0: reg 0x10: [mem 0x00000000-0x7fffffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.498010] pci 0000:00:00.0: reg 0x14: [mem 0x60500000-0x6050ffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.510497] pci 0000:00:00.0: supports D1
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.518377] pci 0000:00:00.0: PME# supported from D0 D1 D3hot
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.530641] pci 0000:00:01.0: [0e8d:0801] type 01 class 0x060400
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.542574] pci 0000:00:01.0: reg 0x10: [mem 0x00000000-0x7fffffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.554956] pci 0000:00:01.0: reg 0x14: [mem 0x60510000-0x6051ffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.567478] pci 0000:00:01.0: supports D1
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.575309] pci 0000:00:01.0: PME# supported from D0 D1 D3hot
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.589010] pci 0000:01:00.0: [14c3:7603] type 00 class 0x028000
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.600926] pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x000fffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.613453] pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.625833] pci 0000:00:00.0: PCI bridge to [bus 01-ff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.636158] pci 0000:00:00.0:   bridge window [io  0x0000-0x0fff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.648213] pci 0000:00:00.0:   bridge window [mem 0x60000000-0x600fffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.661690] pci 0000:00:00.0:   bridge window [mem 0x60100000-0x601fffff pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.676035] pci_bus 0000:01: busn_res: [bus 01-ff] end is updated to 01
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.689481] pci 0000:02:00.0: [14c3:7663] type 00 class 0x000280
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.701378] pci 0000:02:00.0: reg 0x10: [mem 0x00000000-0x000fffff 64bit pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.715672] pci 0000:02:00.0: reg 0x18: [mem 0x00000000-0x00003fff 64bit pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.730021] pci 0000:02:00.0: reg 0x20: [mem 0x00000000-0x00000fff 64bit pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.744472] pci 0000:02:00.0: supports D1 D2
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.752841] pci 0000:02:00.0: PME# supported from D0 D1 D2 D3hot D3cold
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.766003] pci 0000:02:00.0: 2.000 Gb/s available PCIe bandwidth, limited by 2.5 GT/s PCIe x1 link at 0000:00:01.0 (capable of 4.000 Gb/s with 5.0 GT/s PCIe x1 link)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.795860] pci 0000:00:01.0: PCI bridge to [bus 02-ff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.806172] pci 0000:00:01.0:   bridge window [io  0x0000-0x0fff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.818244] pci 0000:00:01.0:   bridge window [mem 0x60200000-0x602fffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.831713] pci 0000:00:01.0:   bridge window [mem 0x60300000-0x604fffff pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.846065] pci_bus 0000:02: busn_res: [bus 02-ff] end is updated to 02
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.859236] pci 0000:00:00.0: BAR 0: no space for [mem size 0x80000000]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.872319] pci 0000:00:00.0: BAR 0: failed to assign [mem size 0x80000000]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.886152] pci 0000:00:01.0: BAR 0: no space for [mem size 0x80000000]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.899274] pci 0000:00:01.0: BAR 0: failed to assign [mem size 0x80000000]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.913107] pci 0000:00:00.0: BAR 8: assigned [mem 0x60000000-0x600fffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.926582] pci 0000:00:00.0: BAR 9: assigned [mem 0x60100000-0x601fffff pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.940931] pci 0000:00:01.0: BAR 8: assigned [mem 0x60200000-0x602fffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.954408] pci 0000:00:01.0: BAR 9: assigned [mem 0x60300000-0x604fffff pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.968743] pci 0000:00:00.0: BAR 1: assigned [mem 0x60500000-0x6050ffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.982233] pci 0000:00:01.0: BAR 1: assigned [mem 0x60510000-0x6051ffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    1.995703] pci 0000:00:00.0: BAR 7: assigned [io  0x0000-0x0fff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.007804] pci 0000:00:01.0: BAR 7: assigned [io  0x1000-0x1fff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.019901] pci 0000:01:00.0: BAR 0: assigned [mem 0x60000000-0x600fffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.033374] pci 0000:00:00.0: PCI bridge to [bus 01]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.043220] pci 0000:00:00.0:   bridge window [io  0x0000-0x0fff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.055316] pci 0000:00:00.0:   bridge window [mem 0x60000000-0x600fffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.068806] pci 0000:00:00.0:   bridge window [mem 0x60100000-0x601fffff pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.083145] pci 0000:02:00.0: BAR 0: assigned [mem 0x60300000-0x603fffff 64bit pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.098533] pci 0000:02:00.0: BAR 2: assigned [mem 0x60400000-0x60403fff 64bit pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.113908] pci 0000:02:00.0: BAR 4: assigned [mem 0x60404000-0x60404fff 64bit pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.129292] pci 0000:00:01.0: PCI bridge to [bus 02]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.139123] pci 0000:00:01.0:   bridge window [io  0x1000-0x1fff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.151219] pci 0000:00:01.0:   bridge window [mem 0x60200000-0x602fffff]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.164697] pci 0000:00:01.0:   bridge window [mem 0x60300000-0x604fffff pref]
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.180697] Serial: 8250/16550 driver, 3 ports, IRQ sharing disabled
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.196039] printk: console [ttyS0] disabled
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.205341] 1e000c00.uartlite: ttyS0 at MMIO 0x1e000c00 (irq = 18, base_baud = 3125000) is a 16550A
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.223339] printk: console [ttyS0] enabled
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.239876] printk: bootconsole [early0] disabled
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.263563] spi-mt7621 1e000b00.spi: sys_freq: 220000000
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.275595] spi-nor spi0.0: en25qx128a (16384 Kbytes)
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.285826] 4 fixed-partitions partitions found on MTD device spi0.0
Thu Jan 30 12:35:29 2025 kern.err kernel: [    2.298567] OF: Bad cell count for /palmbus@1e000000/spi@b00/flash@0/partitions
Thu Jan 30 12:35:29 2025 kern.err kernel: [    2.313361] OF: Bad cell count for /palmbus@1e000000/spi@b00/flash@0/partitions
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.328266] Creating 4 MTD partitions on "spi0.0":
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.337868] 0x000000000000-0x000000030000 : "u-boot"
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.349574] 0x000000030000-0x000000040000 : "u-boot-env"
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.361406] 0x000000040000-0x000000050000 : "radio"
Thu Jan 30 12:35:29 2025 kern.err kernel: [    2.372487] OF: Bad cell count for /palmbus@1e000000/spi@b00/flash@0/partitions
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.387454] 0x000000050000-0x000001000000 : "firmware"
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.399005] 2 uimage-fw partitions found on MTD device firmware
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.410871] Creating 2 MTD partitions on "firmware":
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.420771] 0x000000000000-0x0000003206b5 : "kernel"
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    2.430669] mtd: partition "kernel" doesn't end on an erase/write block -- force read-only
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.448372] 0x0000003206b5-0x000000fb0000 : "rootfs"
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    2.458328] mtd: partition "rootfs" doesn't start on an erase/write block boundary -- force read-only
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.477981] mtd: setting mtd5 (rootfs) as root device
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.488210] 1 squashfs-split partitions found on MTD device rootfs
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    2.500539] 0x000000710000-0x000000fb0000 : "rootfs_data"
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.650258] mt7530-mdio mdio-bus:1f: MT7530 adapts as multi-chip module
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.671533] mtk_soc_eth 1e100000.ethernet eth0: mediatek frame engine at 0xbe100000, irq 19
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.691429] mtk_soc_eth 1e100000.ethernet eth1: mediatek frame engine at 0xbe100000, irq 19
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.709717] i2c_dev: i2c /dev entries driver
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.722233] NET: Registered PF_INET6 protocol family
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.735329] Segment Routing with IPv6
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.742799] In-situ OAM (IOAM) with IPv6
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.750772] NET: Registered PF_PACKET protocol family
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.761240] 8021q: 802.1Q VLAN Support v1.8
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.799138] mt7530-mdio mdio-bus:1f: MT7530 adapts as multi-chip module
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.839487] mt7530-mdio mdio-bus:1f: configuring for fixed/rgmii link mode
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.854629] mt7530-mdio mdio-bus:1f lan1 (uninitialized): PHY [mt7530-0:00] driver [MediaTek MT7530 PHY] (irq=21)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.876841] mt7530-mdio mdio-bus:1f: Link is Up - 1Gbps/Full - flow control rx/tx
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.878930] mt7530-mdio mdio-bus:1f lan2 (uninitialized): PHY [mt7530-0:01] driver [MediaTek MT7530 PHY] (irq=22)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.915223] mt7530-mdio mdio-bus:1f lan3 (uninitialized): PHY [mt7530-0:02] driver [MediaTek MT7530 PHY] (irq=23)
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.938556] mtk_soc_eth 1e100000.ethernet eth0: entered promiscuous mode
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.952129] DSA: tree 0 setup
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.958985] clk: Disabling unused clocks
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.972729] VFS: Mounted root (squashfs filesystem) readonly on device 31:5.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    2.990827] Freeing unused kernel image (initmem) memory: 1240K
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    3.002709] This architecture does not have kernel memory protection.
Thu Jan 30 12:35:29 2025 kern.info kernel: [    3.015558] Run /sbin/init as init process
Thu Jan 30 12:35:29 2025 kern.debug kernel: [    3.023722]   with arguments:
Thu Jan 30 12:35:29 2025 kern.debug kernel: [    3.023730]     /sbin/init
Thu Jan 30 12:35:29 2025 kern.debug kernel: [    3.023737]   with environment:
Thu Jan 30 12:35:29 2025 kern.debug kernel: [    3.023743]     HOME=/
Thu Jan 30 12:35:29 2025 kern.debug kernel: [    3.023748]     TERM=linux
Thu Jan 30 12:35:29 2025 user.info kernel: [    3.468321] init: Console is alive
Thu Jan 30 12:35:29 2025 user.info kernel: [    3.475379] init: - watchdog -
Thu Jan 30 12:35:29 2025 user.info kernel: [    4.316876] kmodloader: loading kernel modules from /etc/modules-boot.d/*
Thu Jan 30 12:35:29 2025 kern.warn kernel: [    4.452086] gpio_button_hotplug: loading out-of-tree module taints kernel.
Thu Jan 30 12:35:29 2025 user.info kernel: [    4.473549] kmodloader: done loading kernel modules from /etc/modules-boot.d/*
Thu Jan 30 12:35:29 2025 user.info kernel: [    4.493966] init: - preinit -
Thu Jan 30 12:35:29 2025 kern.info kernel: [    5.175398] mtk_soc_eth 1e100000.ethernet wan: renamed from eth1
Thu Jan 30 12:35:29 2025 kern.notice kernel: [    6.566972] random: crng init done
Thu Jan 30 12:35:29 2025 kern.info kernel: [    6.981700] mtk_soc_eth 1e100000.ethernet eth0: configuring for fixed/rgmii link mode
Thu Jan 30 12:35:29 2025 kern.info kernel: [    7.001965] mt7530-mdio mdio-bus:1f lan1: configuring for phy/gmii link mode
Thu Jan 30 12:35:29 2025 kern.info kernel: [    7.016140] mtk_soc_eth 1e100000.ethernet eth0: Link is Up - 1Gbps/Full - flow control rx/tx
Thu Jan 30 12:35:29 2025 user.notice kernel: [   11.352620] mount_root: jffs2 not ready yet, using temporary tmpfs overlay
Thu Jan 30 12:35:29 2025 user.warn kernel: [   11.373228] urandom-seed: Seed file not found (/etc/urandom.seed)
Thu Jan 30 12:35:29 2025 user.info kernel: [   11.481505] procd: - early -
Thu Jan 30 12:35:29 2025 user.info kernel: [   11.487535] procd: - watchdog -
Thu Jan 30 12:35:29 2025 user.info kernel: [   12.082259] procd: - watchdog -
Thu Jan 30 12:35:29 2025 user.info kernel: [   12.205645] procd: - ubus -
Thu Jan 30 12:35:29 2025 user.info kernel: [   12.367157] procd: - init -
Thu Jan 30 12:35:29 2025 user.info kernel: [   13.125520] kmodloader: loading kernel modules from /etc/modules.d/*
Thu Jan 30 12:35:29 2025 kern.info kernel: [   13.276663] mtk-eip93 1e004000.crypto: EIP93 Crypto Engine Initialized.
Thu Jan 30 12:35:29 2025 kern.info kernel: [   13.655981] jitterentropy: Initialization failed with host not compliant with requirements: 9
Thu Jan 30 12:35:29 2025 kern.info kernel: [   13.768116] Loading modules backported from Linux version v6.12.6-0-ge9d65b48ce1a
Thu Jan 30 12:35:29 2025 kern.info kernel: [   13.783094] Backport generated by backports.git v6.1.110-1-35-g410656ef04d2
Thu Jan 30 12:35:29 2025 kern.info kernel: [   13.984792] pci 0000:00:00.0: enabling device (0006 -> 0007)
Thu Jan 30 12:35:29 2025 kern.info kernel: [   13.996223] mt7603e 0000:01:00.0: enabling device (0000 -> 0002)
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.008412] mt7603e 0000:01:00.0: ASIC revision: 76030010
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.029420] mt7603e 0000:01:00.0: Firmware Version: ap_pcie
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.040660] mt7603e 0000:01:00.0: Build Time: 20160107100755
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.086961] mt7603e 0000:01:00.0: firmware init done
Thu Jan 30 12:35:29 2025 user.info kernel: [   14.230513] urngd: v1.0.2 started.
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.262965] mt7603e 0000:01:00.0: registering led 'mt76-phy0'
Thu Jan 30 12:35:29 2025 kern.debug kernel: [   14.408457] ieee80211 phy0: Selected rate control algorithm 'minstrel_ht'
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.421983] pci 0000:00:01.0: enabling device (0006 -> 0007)
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.433343] mt7615e 0000:02:00.0: enabling device (0000 -> 0002)
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.459839] mt7615e 0000:02:00.0: registering led 'mt76-phy1'
Thu Jan 30 12:35:29 2025 kern.debug kernel: [   14.561760] ieee80211 phy1: Selected rate control algorithm 'minstrel_ht'
Thu Jan 30 12:35:29 2025 kern.debug kernel: [   14.561833] ieee80211 phy1: copying sband (band 1) due to VHT EXT NSS BW flag
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.635772] PPP generic driver version 2.4.2
Thu Jan 30 12:35:29 2025 kern.info kernel: [   14.647887] NET: Registered PF_PPPOX protocol family
Thu Jan 30 12:35:29 2025 user.info kernel: [   15.192547] kmodloader: done loading kernel modules from /etc/modules.d/*
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.351867] mt7615e 0000:02:00.0: mediatek/mt7663pr2h.bin not found, switching to mediatek/mt7663pr2h_rebb.bin
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.481966] mt7615e 0000:02:00.0: HW/SW Version: 0x65322d31, Build Time: 2009041715da1a1
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.481966]
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.630173] mt7615e 0000:02:00.0: N9 Firmware Version: 7663mp1827, Build Time: 20200904171623
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.647329] mt7615e 0000:02:00.0: Region number: 0x3
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.657240] mt7615e 0000:02:00.0: Parsing tailer Region: 0
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.670852] mt7615e 0000:02:00.0: Region 0, override_addr = 0x00112c00
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.683971] mt7615e 0000:02:00.0: Parsing tailer Region: 1
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.695662] mt7615e 0000:02:00.0: Parsing tailer Region: 2
Thu Jan 30 12:35:29 2025 kern.info kernel: [   15.707011] mt7615e 0000:02:00.0: override_addr = 0x00112c00, option = 3
Thu Jan 30 12:35:30 2025 user.notice dnsmasq: DNS rebinding protection is active, will discard upstream RFC1918 responses!
Thu Jan 30 12:35:30 2025 user.notice dnsmasq: Allowing 127.0.0.0/8 responses
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: started, version 2.90 cachesize 1000
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: DNS service limited to local subnets
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: compile time options: IPv6 GNU-getopt no-DBus UBus no-i18n no-IDN DHCP no-DHCPv6 no-Lua TFTP no-conntrack no-ipset no-nftset no-auth no-cryptohash no-DNSSEC no-ID loop-detect inotify dumpfile
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: UBus support enabled: connected to system bus
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: using only locally-known addresses for test
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: using only locally-known addresses for onion
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: using only locally-known addresses for localhost
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: using only locally-known addresses for local
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: using only locally-known addresses for invalid
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: using only locally-known addresses for bind
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: using only locally-known addresses for lan
Thu Jan 30 12:35:30 2025 daemon.warn dnsmasq[1]: no servers found in /tmp/resolv.conf.d/resolv.conf.auto, will retry
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: read /etc/hosts - 12 names
Thu Jan 30 12:35:30 2025 daemon.info dnsmasq[1]: read /tmp/hosts/dhcp.cfg01411c - 0 names
Thu Jan 30 12:35:43 2025 authpriv.info dropbear[1362]: Not backgrounding
Thu Jan 30 12:35:45 2025 user.notice : Added device handler type: bonding
Thu Jan 30 12:35:45 2025 user.notice : Added device handler type: 8021ad
Thu Jan 30 12:35:45 2025 user.notice : Added device handler type: 8021q
Thu Jan 30 12:35:45 2025 user.notice : Added device handler type: macvlan
Thu Jan 30 12:35:45 2025 user.notice : Added device handler type: veth
Thu Jan 30 12:35:45 2025 user.notice : Added device handler type: bridge
Thu Jan 30 12:35:45 2025 user.notice : Added device handler type: Network device
Thu Jan 30 12:35:45 2025 user.notice : Added device handler type: tunnel
Thu Jan 30 12:35:45 2025 daemon.notice wpa_supplicant[1494]: Successfully initialized wpa_supplicant
Thu Jan 30 12:35:45 2025 daemon.notice procd: /etc/rc.d/S25packet_steering: pid 243's current affinity list: 0-3
Thu Jan 30 12:35:45 2025 daemon.notice procd: /etc/rc.d/S25packet_steering: pid 243's new affinity list: 0
Thu Jan 30 12:35:45 2025 daemon.notice procd: /etc/rc.d/S25packet_steering: pid 244's current affinity list: 0-3
Thu Jan 30 12:35:45 2025 daemon.notice procd: /etc/rc.d/S25packet_steering: pid 244's new affinity list: 0
Thu Jan 30 12:35:46 2025 kern.warn kernel: [   35.975144] jffs2_scan_eraseblock(): End of filesystem marker found at 0x0
Thu Jan 30 12:35:46 2025 kern.warn kernel: [   35.989077] jffs2_build_filesystem(): unlocking the mtd device...
Thu Jan 30 12:35:46 2025 kern.warn kernel: [   35.989103] done.
Thu Jan 30 12:35:47 2025 kern.warn kernel: [   36.005361] jffs2_build_filesystem(): erasing all blocks after the end marker...
Thu Jan 30 12:35:47 2025 kern.info kernel: [   37.537320] mtk_soc_eth 1e100000.ethernet eth0: Link is Down
Thu Jan 30 12:35:47 2025 kern.info kernel: [   37.597405] mtk_soc_eth 1e100000.ethernet eth0: configuring for fixed/rgmii link mode
Thu Jan 30 12:35:47 2025 kern.info kernel: [   37.614092] mtk_soc_eth 1e100000.ethernet eth0: Link is Up - 1Gbps/Full - flow control rx/tx
Thu Jan 30 12:35:47 2025 kern.info kernel: [   37.637365] mt7530-mdio mdio-bus:1f lan1: configuring for phy/gmii link mode
Thu Jan 30 12:35:47 2025 kern.info kernel: [   37.653060] br-lan: port 1(lan1) entered blocking state
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.663624] br-lan: port 1(lan1) entered disabled state
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.674151] mt7530-mdio mdio-bus:1f lan1: entered allmulticast mode
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.686698] mtk_soc_eth 1e100000.ethernet eth0: entered allmulticast mode
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.703188] mt7530-mdio mdio-bus:1f lan1: entered promiscuous mode
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'lan' is enabled
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'lan' is setting up now
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'lan' is now up
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.737277] mt7530-mdio mdio-bus:1f lan2: configuring for phy/gmii link mode
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.752458] br-lan: port 2(lan2) entered blocking state
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.763083] br-lan: port 2(lan2) entered disabled state
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.773793] mt7530-mdio mdio-bus:1f lan2: entered allmulticast mode
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.789076] mt7530-mdio mdio-bus:1f lan2: entered promiscuous mode
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.811571] mt7530-mdio mdio-bus:1f lan3: configuring for phy/gmii link mode
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.826707] br-lan: port 3(lan3) entered blocking state
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.837285] br-lan: port 3(lan3) entered disabled state
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.847840] mt7530-mdio mdio-bus:1f lan3: entered allmulticast mode
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.862978] mt7530-mdio mdio-bus:1f lan3: entered promiscuous mode
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'loopback' is enabled
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'loopback' is setting up now
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'loopback' is now up
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.895727] mtk_soc_eth 1e100000.ethernet wan: PHY [mt7530-0:04] driver [MediaTek MT7530 PHY] (irq=POLL)
Thu Jan 30 12:35:48 2025 kern.info kernel: [   37.914877] mtk_soc_eth 1e100000.ethernet wan: configuring for phy/rgmii link mode
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'wan' is enabled
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'wan6' is enabled
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Network device 'eth0' link is up
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Network device 'lo' link is up
Thu Jan 30 12:35:48 2025 daemon.notice netifd: Interface 'loopback' has link connectivity
Thu Jan 30 12:35:48 2025 user.notice firewall: Reloading firewall due to ifup of lan (br-lan)
Thu Jan 30 12:35:51 2025 kern.info kernel: [   40.867666] mt7530-mdio mdio-bus:1f lan2: Link is Up - 1Gbps/Full - flow control rx/tx
Thu Jan 30 12:35:51 2025 kern.info kernel: [   40.867741] br-lan: port 2(lan2) entered blocking state
Thu Jan 30 12:35:51 2025 kern.info kernel: [   40.893966] br-lan: port 2(lan2) entered forwarding state
Thu Jan 30 12:35:51 2025 daemon.notice netifd: Network device 'lan2' link is up
Thu Jan 30 12:35:51 2025 daemon.notice netifd: bridge 'br-lan' link is up
Thu Jan 30 12:35:51 2025 daemon.notice netifd: Interface 'lan' has link connectivity
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: exiting on receipt of SIGTERM
Thu Jan 30 12:36:00 2025 daemon.err procd: Got unexpected signal 1
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: started, version 2.90 cachesize 1000
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: DNS service limited to local subnets
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: compile time options: IPv6 GNU-getopt no-DBus UBus no-i18n no-IDN DHCP no-DHCPv6 no-Lua TFTP no-conntrack no-ipset no-nftset no-auth no-cryptohash no-DNSSEC no-ID loop-detect inotify dumpfile
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: UBus support enabled: connected to system bus
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq-dhcp[1]: DHCP, IP range 192.168.1.100 -- 192.168.1.249, lease time 12h
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: using only locally-known addresses for test
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: using only locally-known addresses for onion
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: using only locally-known addresses for localhost
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: using only locally-known addresses for local
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: using only locally-known addresses for invalid
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: using only locally-known addresses for bind
Thu Jan 30 12:36:00 2025 daemon.info dnsmasq[1]: using only locally-known addresses for lan
Thu Jan 30 12:36:00 2025 daemon.warn dnsmasq[1]: no servers found in /tmp/resolv.conf.d/resolv.conf.auto, will retry


</pre>
</details>

namiltd commented:
Thank you very much :)
